v2.0/roles.md

```diff
@@ -0,0 +1,201 @@
+---
+title: Manage Roles
+summary: Roles are SQL groups that contain any number of users and roles as members.
+toc: false
+---
+
+<span class="version-tag">New in v2.0:</span> Roles are SQL groups that contain any number of users and roles as members. To create and manage your cluster's roles, use the following statements:
+
+- [`CREATE ROLE` (Enterprise)](create-role.html)
+- [`DROP ROLE` (Enterprise)](drop-role.html)
+- [`GRANT [role]` (Enterprise)](grant.html)
+- [`REVOKE [role]` (Enterprise)](revoke.html)
+- [`GRANT [privilege]`](grant.html)
+- [`REVOKE [privilege]`](revoke.html)
+- [`SHOW ROLES`](show-roles.html)
+
+<div id="toc"></div>
+
+## Terminology
+
+To get started, basic role terminology is outlined below:
+
+Term | Description
+-----|------------
+Role | A group containing any number of [users](create-and-manage-users.html) or other roles.
+Role admin | A member of the role that's allowed to modify role membership. To create a role admin, use [`WITH ADMIN OPTION`](grant.html).
+Superuser / Admin | A member of the `admin` role. Only superusers can `CREATE ROLE` or `DROP ROLE`. The `admin` role is created by default and cannot be dropped.
+`root` | A user that exists by default as a member of the `admin` role. The `root` role must always be a member of `admin`.
+Inherit | The behavior that grants a role's privileges to its members.
+`∈` | In the context of roles, `∈` translates to "is a member of."<br><br>Example: `A ∈ B` denotes that user or role `A` is a member of role `B`.
```

Comment:
I used this notation in the RFC because I didn't feel like typing `is a member of` over and over again. Do we want this type of symbol in the docs, or do we want to stick to plain english?